### PR TITLE
Create Source Code, Execute, Queue New Terms

### PR DESCRIPTION
**Pull request by sandyavila**
Source code, is a human-readable text written in a specific programming language created by a programmer with the goal of setting exact rules for a computer-executable application.

Execute, is the process of running a computer software program, script, or command by reading and acting on the instructions converted into machine language.

Queue , is a data structure in which elements are removed in the same order they are entered (FIFO)- first-in first-out abstract data type used in computing.

Fill in each of the sections (using NA for those that are not applicable).

## Author: 

Sandy Avila

## Language: 

English

## Terms defined:

- Source Code
- Execute
- Queue

### CODE REVIEW

**Comment by baileythegreen** [comment]
Hi @sandyavila,

So, a few comments (in case you plan to submit more things here—and I hope you do).

1. You should fill out the sections of the PR template, so that it looks something like this (no need to include the definitions here). :



> Fill in each of the sections (using NA for those that are not applicable).
> 
> ## Author: 
> 
> - @sandyavila 
> 
> ## Language: 
> 
> - English
> 
> ## Terms defined:
> 
> - source code
> - queue
> - execute


1. You haven't actually committed any changes. In order to do so you need to edit the `glossary.yml` file, either in the browser (which will open it in a new branch) or locally on your computer, using your favourite text editor. Then you have to commit the changes (in the browser, this involves clicking on 'Commit' at the bottom of the page; locally it's more involved and depends on if you are using the command line or the desktop application).

You can see no changes have been committed by looking at the 'Files changed' tab of this PR. It should look like this: https://github.com/carpentries/glosario/pull/282/files.

This link might help you get this working (clarifying how to commit and how to submit a pull request): https://guides.github.com/activities/hello-world/

**Comment by sandyavila** [comment]
Thank you, Bailey for the updated details. I am pretty new to using GitHub and officially with creating Pull Requests. 😊

I had a feeling things weren’t posted correctly. I will go through this later today and fix it all and send any other questions or concerns I have in the process. But I hope that with the details you provide here, I can get it right this time. I very much appreciate your quick input and response!

Have a great Monday!

-Sandy

**Comment by sandyavila** [comment]
Hi again, Bailey.

I am soo struggling through getting these terms to commit properly. I see the file and try to edit it but that option doesn't show up for me as an available option. I was able to put my code into comment which I hope you can see.

Any other kind of help you can provide would be greatly appreciated. I can meet you via Zoom to walk through steps if possible and thanks for your help in advance.

-Sandy

**Comment by baileythegreen** [comment]
@sandyavila It looks like whatever you did last has managed to put the lines into the file successfully.

*However*, this is a glossary, and the entries need to be put into the file in alphabetical order. Please follow the same procedure you just did to get it to work, but cutting the lines from the end of the file and pasting them into the appropriate places.

**Comment by sandyavila** [comment]
Got it! So that is all I had to do!? I am not sure how I figured it out but I am glad it worked. I will put them in alphabetical order. Thanks, Bailey!

**Comment by baileythegreen** [comment]
To be clear, once you have done that, I will then actually review the entries, and some code checks will be performed. Once the code checks clear, and I approve the content, it will be merged, but I sometimes request changes to the definitions, or to the formatting so that the code checks will pass.

**Comment by sandyavila** [comment]
That is totally fine! I will look out for those messages and correct as requested. Thanks for ALL your help. I am fixing the alphabetical issues right now.

-Sandy

File: glossary.yml [hunk 1]
```diff
@@ -4805,7 +4811,13 @@
       The portion of a [URL](#url) after the question mark `?` that specifies extra
       parameters for the [HTTP request](#http_request) as name-value pairs.
 
-
+- slug: queue
+  en:
+    term: "queue"
+    def: >
+      Is a data structure in which elements are removed in the same order they are entered 
+     (FIFO)- first-in first-out abstract data type used in computing.
```
Comment (baileythegreen) [inline glossary.yml:4819]:
```suggestion
      (FIFO)- first-in first-out abstract data type used in computing.
```

File: glossary.yml [hunk 2]
```diff
@@ -5580,6 +5592,12 @@
       A set of programs that are built, tested, and distributed as a collection so
       that they can run together.
 
+- slug: soure code
+  en:
+    term: "source code"
+    def: >
+      Is a human-readable text written in a specific programming language created by a programmer 
+      with the goal of setting exact rules for a computer-executable application.
```
Comment (baileythegreen) [inline glossary.yml:5600]:
```suggestion
      Human-readable text written in a programming language 
      with the goal of setting exact rules for a computer-executable application.
```

I don't agree with the

> goal of setting exact rules for

bit; that sounds more like it refers to parameter settings than source code as a whole. The concept of source code as a series of actions is probably more useful. Also, the definition should reference the difference between source code and compiled code explicitly; likely in a second sentence. Right now, this is only implied by the words 'human-readable'.

**Review by baileythegreen — CHANGES_REQUESTED** [review]

File: glossary.yml [hunk 3]
```diff
@@ -5580,6 +5593,12 @@
       A set of programs that are built, tested, and distributed as a collection so
       that they can run together.
 
+- slug: soure code
+  en:
+    term: "source code"
+    def: >
+      Human-readable text written in a programming language 
+      with the goal of setting exact rules for a computer-executable application.
```
Comment (baileythegreen) [inline glossary.yml:5601]: I don't agree with the

> goal of setting exact rules for

bit; that sounds more like it refers to parameter settings than source code as a whole. The concept of source code as a series of actions is probably more useful. Also, the definition should reference the difference between source code and compiled code explicitly; likely in a second sentence. Right now, this is only implied by the words 'human-readable'.

**Review by baileythegreen — CHANGES_REQUESTED** [review]
See inline comment.

**Comment by elletjies** [comment]
Closing this pull request since the last activity took place in April 2021. Please feel free to reopen in the future to make the necessary changed.

**Pull request closed without merging**